docs/advanced.md

```diff
@@ -0,0 +1,71 @@
+
+`script GetPlayerArray()[0].SetPersistentVar("gen", insertGenCount)` this command sets the Generation level of the player. You want to replace `insertGenCount` with the number you want.
+
+`script GetPlayerArray()[0].SetPersistentVar("xp", insertXpCount)` this command sets the xp count of the player (meaning, the amount of kills required per level in order to level up. Make sure to set this to less than 472 as to not encounter issues). You want to replace `insertXpCount` with the number you want.
+
+If you experience strange issues after using these, you probably set something too high, and should follow the [resetting levels wiki section.](installing-northstar/troubleshooting.md#i-used-a-command-to-set-my-playergun-xp-level-and-i-set-it-too-high-so-now-my-game-crashes-when-trying-to-join-multiplayer)
+
+## Playing Vanilla via NorthstarLauncher <a href="#vanilla-on-northstar" id="vanilla-on-northstar"></a>
+
+It is recommended that you [set up a vanilla profile](advanced.md#profiles) instead of disabling all of your mods and using `-norestrictservercommands` as a launch option, however if you wish to only play vanilla using this method and would rather not set up a profile, you can do the following.
+
+The reason behind doing this is that Northstar has several security fixes that Vanilla does not have, however these are not *neccesary*. The odds you get hacked playing vanilla are close to zero, and there are no reports of people being genuinely hacked by playing Vanilla, the worst reports being that they've been taken offline temporarily due to DDOS attacks.
+
+This method assumes you're launching Northstar via Titanfall 2 on EA/Steam/Origin using launch options
+
+1. Go to the main menu of Northstar 
+2. Click `"Mods"` on the bottom of the menu
+3. Disable **ALL** of your mods
+4. Close Northstar
+5. Add `-norestrictservercommands` and `-northstar` as a [launch option](installing-northstar/troubleshooting.md#launch-opts)
+6. Launch Titanfall 2 and you should be able to play on vanilla servers using Northstar's security fixes
+
+**TO PLAY ON NORTHSTAR AGAIN**
+
+1. Go to your [titanfall2 directory](installing-northstar/troubleshooting.md#game-location)
+2. Open your `R2Northstar` folder
+3. Delete `enabledmods.json`
+4. Launch Northstar
+
+## Resetting Master Server
+
+**Only do this if you have changed your master server!**
+
+  * Open `R2Northstar\Northstar.CustomServers\mod\cfg\autoexec_ns_server.cfg`
+  * Change the `ns_masterserver_hostname` line to `ns_masterserver_hostname "https://northstar.tf`
+  * Open `R2Northstar\Northstar.Client\mod\cfg\autoexec_ns_client.cfg
+  * Change the `ns_masterserver_hostname` line to `ns_masterserver_hostname "https://northstar.tf`
+  * Launch Northstar, and your master server should be properly reset
```

Comment:
There wasn't a need to change master server in a while now and AFAIK we're also planning to even move master server to `api.northstar.tf`, so I'd just remove this section as making sure it's update is more hassle than it's worth ^^

Reply:
I remember this being used once recently but yeah I imagined use cade would be low. Will remove